Add notes from to selfAuth with Azure docs (#4835)
flyteorg/flyte: docs/deployment/configuration/auth_setup.rst

           # Use the non-encoded version of the random password 
             clientSecret: "<your-random-password>" 
             clientId: flytepropeller

.. note::

>    For `multi-cluster deployments <https://docs.flyte.org/en/latest/deployment/deployment/multicluster.html>`__, this secret definition has to be added to the `values-dataplane.yaml` file.

Reviewer: ```suggestion
   For `multi-cluster deployments <https://docs.flyte.org/en/latest/deployment/deployment/multicluster.html>`__ running `flytepropeller` in the control plane cluster, you must add this secret definition to the `values-dataplane.yaml` file.
```

Author: I'm trying not to get lost in grammar :) The idea is: in every cluster where propeller is running you need that secret (not only environments where propeller is running on the control plane cluster).
Propeller in the control plane cluster is optional, for the data plane is a must.

Let me know if it's clear or not

Reviewer: Gotcha, then I would suggest:
```
For `multi-cluster deployments <https://docs.flyte.org/en/latest/deployment/deployment/multicluster.html>`__, you must add this secret definition to the `values-dataplane.yaml` file.
```

>    There's no need to create this secret in the control plane cluster if you're not running `flytepropeller` there.

Reviewer: ```suggestion
   If you are not running `flytepropeller` in the control plane cluster, you do not need to create this secret there.
```

      8. Save and exit your editor.

      9. Upgrade your Helm release with the new configuration:

      .. prompt:: bash $

         helm upgrade <release-name> flyteorg/flyte-binary -n <your-namespace> --values <your-values-file>.yaml

>       10. Verify that the `flytepropeller`, `flytescheduler` and `flyteadmin` Pods are restarted and Running: 

Reviewer 2: What's the reason to remove the instructions on how to restart the deployments (`kubectl rollout restart ...`) in the docs?

>       10. Verify that the `flytepropeller`, `flytescheduler` and `flyteadmin` Pods are restarted and Running: 

Reviewer: I recommend not capitalizing "Running".
